src/components/common/Stepper.vue

```diff
@@ -38,20 +38,19 @@
 
```

Comment:
You can fix the lint warning below by moving the v-for to the div (and then you can remove the template).

Reply:
Thanks for the fix.